lib/webtask.js

```diff
@@ -708,6 +756,66 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function validate_auth_token(auth) {
+            var auth_token;
+            var decoded_auth_token;
+            var userinfo;
+            var aud_should_be;
+            var secret = auth.secret || ctx.config.default_auth0_client_secret.current;
```

Comment:
The problem at the moment here is that we want to have a shared Auth0 account, so that someone can run `wt create my-webtask.js --auth0 @company.com` and scope the webtask (a la sharelock). Putting this secret any higher on the chain right now puts it in userland and exposed. I will have a think about this.